rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -377,6 +399,13 @@ def mock_requests_get(url: str, timeout: int = 0, headers: dict | None = None):
         'missing_decoder': True,
         'icon_url': 'https://raw.githubusercontent.com/rotki/data/develop/airdrops/icons/degen.svg',
     }
+    assert data[TEST_ADDR2]['eigen'] == {
+        'amount': '10',
+        'asset': Asset('EIGEN_TOKEN_PRE_RELEASE'),
+        'link': 'https://www.degen.tips/airdrop2/season1',
```

Comment:
where is this link coming from? it's same as the degen above, something seems off